release-team/role-handbooks/communications/README.md

```diff
@@ -43,11 +44,59 @@ The list has to be rotated/actively managed every cycle. Submit a PR to update [
 
 ### Slack Channel
 
-There is a channel on the Kubernetes Slack workspace, `release-comms`, which is used by the communications release team to coordinate their efforts. If you're on the communications team, or applying to be, then it would be advantageous to follow along with the conversations.
+There is a channel on the Kubernetes Slack workspace, `#release-comms`, which is used by the communications release team to coordinate their efforts. If you're on the communications team, or applying to be, then it would be advantageous to follow along with the conversations. The Communications Coordinator (often referred to as the "Comms Lead") should post a status here at a regular cadence to keep release team members and other stakeholders informed.
+
+## Release deliverables
+
+Throughout the release cycle, the main Communications deliverables include:
+* **Authoring the Kubernetes release blog.** The Communications team writes the release blog, which is the official announcement of the Kubernetes release.
+* **Coordination and support of the feature blog series.** SIGs opt in to author feature blogs for the release. These blogs are written by technical owners, and the Communications team supports authors from the early stages through facilitating editorial and tech reviews as well as publication.
+* **Coordination and support of an optional Deprecations and Removals blog.** Depending upon the content of a given release, it may be necessary to prepare the community for upcoming deprecations and removals. This is decided per release cycle around the time of Code Freeze.
```

Comment:
Just a few thoughts around the deprecations and comms - I'm not sure if its worth calling out explicitly, but around those I would engage with the contributor comms team and the CNCF (the contributor comms team might be able to facilitate that). For larger deprecations we really need to make sure the information is socialized and spread as much as possible and make sure we can preempt any sort of FUD.

Reply:
It might be good to add this in general (connecting the comms teams). Can you suggest the best ongoing point of contact for that (slack, email, attending a meeting, etc)? I will add something in both here, and maybe a note to reach out and intro at the cycle startup as well.

Reply:
We can also add this later. I'd really like to get the bulk of this merged if possible, as the release is starting up. Would we be ok merging it as it is now?

Reply:
The current version looks good to me. I would be comfortable shipping this now and amending it later as needed.

Reply:
I think it would be OK to do a follow up with those contacts. @mickeyboxell can you take an action (either for you or one of your shadows) to make that update?

Reply:
Yep can do.